### PR TITLE
Rpc autocomlte

### PR DESCRIPTION
**Pull request by Adam-Goldman7**
sry for a too big PR

will split it next time!

### CODE REVIEW

File: client/containers/RpcPage/index.js [hunk 1]
```diff
@@ -32,12 +35,13 @@ function mapStateToProps (state) {
 
 function mapDispatchToProps (dispatch) {
   return {
-    actions: bindActionCreators(RpcActions, dispatch)
+    actions: bindActionCreators(_.extend(RpcActions, {addToast: addToast}), dispatch)
```
Comment (tomusdrw) [inline client/containers/RpcPage/index.js]: Should be `_.extend({}, RpcActions, {addToast})`

File: client/index.html [hunk 2]
```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,300,500' rel='stylesheet' type='text/css'>
```
Comment (tomusdrw) [inline client/index.html]: Can we store the font locally?

File: client/components/RpcDocs/index.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    );
+  }
+
+  handleToggle (idx) {
+    const {name} = rpcMethods[idx];
+    if (name === this.props.rpc.selectedDoc) {
+      return;
+    }
+
+    this.setState({selected: name});
```
Comment (tomusdrw) [inline client/components/RpcDocs/index.js]: Why do we need state and action?

File: client/containers/RpcPage/index.js (hunk 1 above)
Comment (Adam-Goldman7) [inline client/containers/RpcPage/index.js]: y the empty object at first?
to not mutate RpcActions?

Comment (tomusdrw) [inline client/containers/RpcPage/index.js]: Yup.

File: client/components/RpcDocs/index.js (hunk 3 above)
Comment (Adam-Goldman7) [inline client/components/RpcDocs/index.js]: state is needed for the SelectableList
action is for remembering which docs was chosen when navigating

think it's unneccesary?

**Pull request merged**